spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
Spam spam spam :-)

Because there is no sliver bulltet rule, if the package is not lazy loaded then putting stuff in `init` or `config` may be wrongly interpreted depending of what the user understood about `use-package`. This is a flaw in the design but we can live with it.